swift/IceGrid/simple/Sources/Server/main.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+import Foundation
+import Ice
+
+class HelloI: Hello {
+    private var name: String
+
+    init(name: String) {
+        self.name = name
+    }
+
+    func sayHello(current _: Current) throws {
+        print(name + " says Hello World!")
+    }
+
+    func shutdown(current: Current) throws {
+        print(name + " shutting down...")
+        current.adapter?.getCommunicator().shutdown()
```

Comment:
Should it be just `current.adapter.getCommunicator()` or `current.adapter!.getCommunicator()` here? current.adapter should never be nil when shutdown is called by the Ice runtime.